14/umbraco-cms/customizing/extending-overview/extension-types/menu.md

```diff
@@ -4,37 +4,173 @@
 This page is a work in progress and may undergo further revisions, updates, or amendments. The information contained herein is subject to change without notice.
 {% endhint %}
 
-## Menu
-
 <figure><img src="../../../.gitbook/assets/menu.png" alt="" width="250"><figcaption><p>Menu</p></figcaption></figure>
 
-```typescript
+**JSON Manifest:**
```

Comment:
The two code snippets here - what exactly to they show? 😅 
Would be good to add a little text explaining 🙌